bread/layout/components/form.py

```diff
@@ -162,23 +167,72 @@ def render(self, context):
                 ),
                 id=f"formset_{formset.prefix}_container",
             ),
-            hg.DIV(
-                Form(formset.empty_form, *self, standalone=False),
+            # empty form as template for new entries, the script works very well for this
+            # since we a single need raw, unescaped HTML string
+            hg.SCRIPT(
+                mark_safe(
+                    hg.render(Form(formset.empty_form, *self, standalone=False), {})
+                ),
                 id=f"empty_{ formset.prefix }_form",
-                _class="template-form",
-                style="display:none;",
+                type="text/plain",
             ),
             hg.SCRIPT(
                 mark_safe(
                     f"""document.addEventListener("DOMContentLoaded", e => init_formset("{ formset.prefix }"));"""
                 )
             ),
-        )
-        yield from skeleton.render(context)
+        ).render(context)
 
     def __repr__(self):
         return f"Formset({self.fieldname}, {self.formsetfactory_kwargs})"
 
+    @staticmethod
+    def as_datatable(
+        fieldname,
+        inlinefieldnames,
+        formname="form",
+        title=None,
+        **formsetfactory_kwargs,
+    ):
+        """
+        :param str fieldname: The fieldname which should be used for an formset, in general a n-to-many field
+        :param list inlinefieldnames: A list of strings or objects. Strings are converted to DataTableColumn, objects are passed on as they are
+        :param str formname: Name of the surounding django-form object in the context
+        :param str title: Datatable title, automatically generated from form if None
+        :return: A datatable with inline-editing capabilities
+        :rtype: hg.HTMLElement
```

Comment:
Because if we wrap with toolbar it is hg.DIV and not DataTable. I guess we could use hg.HTMLTag (which I had first) but it seems to be less clear what you actually get...